global_variable_interface.m

```diff
@@ -0,0 +1,10 @@
+// Global Variable Implementation
+// 
+//
+// IDECodeSnippetCompletionPrefix: vokGlobalVariableImplementation
+// IDECodeSnippetCompletionScopes: [TopLevel]
+// IDECodeSnippetIdentifier: DCB5228E-34F4-4453-BF27-0A022BFF403E
+// IDECodeSnippetLanguage: Xcode.SourceCodeLanguage.Objective-C
+// IDECodeSnippetUserSnippet: 1
+// IDECodeSnippetVersion: 2
+<#Type#> *const <#XXXGlobalVariableName#> = <#Value#>;
```

Comment:
Absolutely I can add some more documentation here and create a new one without the pointer for non-object variables.